test/mongodb.test.js

```diff
@@ -2445,6 +2472,58 @@ describe('mongodb connector', function() {
       });
     });
 
+    it('updateOrCreate should convert geopoint to geojson', function(done) {
```

Comment:
@alexandreferreira I ran the tests without your patch/fix and this test was able to run successfully. Is it an additional test for coverage that is not associated with your fix like https://github.com/strongloop/loopback-connector-mongodb/pull/522/files#diff-f44d7a6206dcaa5ddd05b31118b5c2beR2007?

Reply:
I.e. if this is meant to be covering your changes in `updateOrCreate` method, then it's still passing without the changes and we need to make sure it fails without them to catch regressions in the future.

Reply:
@b-admike I'll check this test